src/components/Form/Form.js

```diff
@@ -4,32 +4,60 @@ import {useRouter} from 'next/router';
 import styled from 'styled-components';
 
 import useStore from '../../Hooks/useStore';
+import {pxToRem} from '../../utils/unit';
 import StyledButton from '../Button/StyledButton';
 import Headline2 from '../Headline/Headline2';
-
 export default function Form() {
 	const addEntry = useStore(state => state.addEntry);
 	const router = useRouter();
-
 	function handleSubmit(event) {
 		event.preventDefault();
-
 		const formData = new FormData(event.target);
-		const {firstEntry, secondEntry, thirdEntry} = Object.fromEntries(formData);
-
+		const {firstEntry, secondEntry, thirdEntry, mood} = Object.fromEntries(formData);
 		event.target.reset();
-		const entry = {id: nanoid(), first: firstEntry, second: secondEntry, third: thirdEntry};
+		const entry = {
+			id: nanoid(),
+			first: firstEntry,
+			second: secondEntry,
+			third: thirdEntry,
+			mood: mood,
+		};
 		addEntry(entry);
-
 		router.push('./diary');
 	}
 	const DynamicWrapper = dynamic(() => import('../styledWrapper'), {
 		ssr: false,
 	});
+
 	return (
 		<DynamicWrapper>
-			<Headline2 />
 			<form onSubmit={handleSubmit} autoComplete="off">
+				<Styledh2>How did you feel today?</Styledh2>
+				<StyledDiv>
```

Comment:
Anstatt alle Radio Buttons einzeln in ein div zu packen , könntest du eine styled Fieldset machen, damit es auch semantisch korrekt ist

Reply:
Thanks for your input Petros, I tried to do it with only one fieldset around everything but I could not style it the way I wanted to. So I would love to leave it that way (however I put a fieldset around everything)